components/scripts/lib/cli-parsers/gradle/03-cli-parser.m4

```diff
@@ -2,6 +2,7 @@
 # shellcheck disable=SC2034  # It is common for variables in this auto-generated file to go unused
 # Created by argbash-init v2.10.0
 # ARG_OPTIONAL_BOOLEAN([build-scan-publishing],[],[],[on])
+# ARG_OPTIONAL_BOOLEAN([disable-build-scan-publishing],[x])
```

Comment:
This is a workaround for a limitation with how Argbash handles boolean CLI switches. Basically, there is no way to set a shorthand for a boolean CLI switch to set the its value to "off".

The actual --disable-build-scan-publishing CLI argument won't be presented to users, but if a user happens to use it, it will do the right thing. There is some potentially odd behavior if both flags are used at the same time, but this seems unlikely (since users won't know there are multiple flags unless they inspect the script source code).

Reply:
Thanks for the explanation.